agent/pkg/routes/config_routes.go

```diff
@@ -1,15 +1,16 @@
 package routes
 
 import (
-	"github.com/gin-gonic/gin"
 	"mizuserver/pkg/controllers"
 	"mizuserver/pkg/middlewares"
+
+	"github.com/gin-gonic/gin"
 )
 
 func ConfigRoutes(ginApp *gin.Engine) {
 	routeGroup := ginApp.Group("/config")
 	routeGroup.Use(middlewares.RequiresAuth())
 
-	routeGroup.POST("/tapConfig", controllers.PostTapConfig)
+	routeGroup.POST("/tapConfig", middlewares.RequiresAdmin(), controllers.PostTapConfig)
```

Comment:
should we rename it to just "tap"? so there will be no duplicate "config" in endpoint:
/config/tapConfig -> /config/tap

Reply:
another thought we should rename it to tappedNamespace (we have different "tap" endpoints some for tapPods status, some for tapper status..)

Reply:
renamed to `/tap`